magefiles/go.mod

```diff
@@ -0,0 +1,18 @@
+module botkube.io/tools
```

Comment:
Please also update main README to make sure Mage is mentioned as a prerequisite. Maybe it would be also good to mention `mage -l` command to run to see all possible targets.